apps/twitch-bot/internal/command/command.go

```diff
@@ -10,12 +10,15 @@ import (
 	"github.com/senchabot-opensource/monorepo/apps/twitch-bot/client"
 	"github.com/senchabot-opensource/monorepo/apps/twitch-bot/internal/command/helpers"
 	"github.com/senchabot-opensource/monorepo/apps/twitch-bot/internal/service"
+	"github.com/senchabot-opensource/monorepo/packages/gosenchabot/models"
 )
 
+type commandFuncReturn map[string]func(context context.Context, message twitch.PrivateMessage, commandName string, params []string) (*models.CommandResponse, error)
```

Comment:
If the `func(context context.Context, message twitch.PrivateMessage, commandName string, params []string) (*models.CommandResponse, error)` is used multiple times other than map[string]func(...)... type, we can divide it into a different type.

Following type declarations can be more readable.
type CommandFunction func(context context.Context, message twitch.PrivateMessage, commandName string, params []string) (*models.CommandResponse, error)

type CommandsMap map[string]CommandFunction